packages/suite/src/components/firmware/Initial.tsx

```diff
@@ -39,7 +39,7 @@ const ChangesSummary = styled.div`
     text-align: left;
     background-color: ${colors.BLACK98};
     border-radius: 8px;
-    padding: 20px;
```

Comment:
as i remember correctly we've agreed at some to use values divided by 4 (so 24 in this case and 4px in ChangesLi case)

Reply:
Did you mean divisible by 8? Because 20 is divisible by 4

Reply:
new one is `padding: 20px 25px;` (idk why github is showing this comment in the "old file" since i've added this comment in "new file")

Reply:
new value is `padding: 20px 25px;` (idk why github is showing this comment in the "old file" since i've added this comment in "new file")

Reply:
Ah, sorry